includes/apple-exporter/class-markdown.php

```diff
@@ -82,170 +62,190 @@ private function parse_nodes( $nodes ) {
 	/**
 	 * Parse an individual node for markdown.
 	 *
-	 * @param DomNode $node
-	 * @return string
+	 * @param DOMElement $node The node to process.
+	 *
 	 * @access private
+	 * @return string The processed content, converted to a Markdown string.
 	 */
-	private function parse_node( $node ) {
+	private function _parse_node( $node ) {
 		switch ( $node->nodeName ) {
-		case 'strong':
-			return $this->parse_strong_node( $node );
-		case  'i':
-		case 'em':
-			return $this->parse_emphasis_node( $node );
-		case 'br':
-			return $this->parse_linebreak_node( $node );
-		case 'p':
-			return $this->parse_paragraph_node( $node );
-		case 'a':
-			return $this->parse_hyperlink_node( $node );
-		case 'ul':
-			return $this->parse_unordered_list_node( $node );
-		case 'ol':
-			return $this->parse_ordered_list_node( $node );
-		case 'li':
-			return $this->parse_list_item_node( $node );
-		case 'h1':
-		case 'h2':
-		case 'h3':
-		case 'h4':
-		case 'h5':
-		case 'h6':
-			return $this->parse_heading_node( $node );
-		case '#comment':
-			return '';
-		case '#text':
-		default:
-			return $this->parse_text_node( $node );
+			case 'strong':
+				return $this->_parse_node_strong( $node );
+			case  'i':
+			case 'em':
+				return $this->_parse_node_emphasis( $node );
+			case 'br':
+				return '  ' . "\n";
```

Comment:
This was in the code previously (I just refactored it) so I believe it is necessary, but can check.